allure-web/src/components/status-toggle/StatusToggleView.js

```diff
@@ -40,7 +40,7 @@ class StatusToggleView extends PopoverView {
     }
 
     serializeData() {
-        const statuses = settings.get(this.statusesKey);
+        const statuses = settings.get(this.statusesKey) || settings.get('visibleStatuses');
```

Comment:
Well, we have repeated that code so many times.
Maybe it would be better to create a new method `getVisibleStatuses(statusesKey)` in the `SettingsModel`?